ex3/ex3.c

```diff
@@ -5,10 +5,34 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
+#include <sys/types.h>
+#include <sys/wait.h>
 
-int main(int argc, char* argv[])
+int main(int argc, char *argv[])
 {
-    // Your code here
+    pid_t FPID = fork();
+    pid_t PID = getpid();
+    printf("Starting... %d", PID);
+    pid_t PPID = getppid();
+
+    FILE *fp;
+    char file[] = "text.txt";
+    fp = fopen(file, "a");
+
+    if (fp != NULL)
+    {
+        if (FPID != 0)
+        {
+            wait(NULL);
+            fprintf(fp, "Goodbye\n", PID, PPID, FPID);
```

Comment:
Too many arguments for the given string

Reply:
Could just be `printf("Goodbye\n");`

Reply:
I was told to just use this as a scratch pad to learn the api. Noone mentioned any limits to testing args.

Reply:
Absolutely! You're free to use as many arguments as you'd like, but you need to provide the % format specifiers for each argument.